mm2src/coins/utxo.rs

```diff
@@ -599,6 +602,9 @@ pub struct UtxoCoinConf {
     pub checksum_type: ChecksumType,
     /// Fork id used in sighash
     pub fork_id: u32,
+    /// A CAIP-2 compliant chain ID. This is used to identify the UTXO chain in WalletConnect and other cross-chain protocols.
+    /// https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-4.md
+    pub chain_id: Option<WcChainId>,
```

Comment:
Let's use permalink.

```suggestion
    /// A CAIP-2 compliant chain ID. This is used to identify the UTXO chain in WalletConnect and other cross-chain protocols.
    /// https://github.com/ChainAgnostic/CAIPs/blob/9516a2c0b26223d98a342/CAIPs/caip-4.md
    pub chain_id: Option<WcChainId>,
```

Reply:
[6248a49](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2566/commits/6248a491d3fabad0429199ea0136acc54852d8e9)